src/Mars-Gtk/GtkWindow.class.st

```diff
@@ -136,6 +136,13 @@ GtkWindow >> getPositionX: root_x y: root_y [
 	^ self ffiCall: #(void gtk_window_get_position (self, gint *root_x, gint *root_y))
 ]
 
+{ #category : #geometry }
+GtkWindow >> getScaleFactor [
```

Comment:
same here, just `scaleFactor`. 
The logic is that public api should have pharo standards (yes, there are methods called getX, but you will see they are all on private protocol and are used by other methods with proper name)

Reply:
done